sdk/identity/azure-identity/azure/identity/aio/__init__.py

```diff
@@ -1,20 +1,33 @@
-# -------------------------------------------------------------------------
+# ------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See LICENSE.txt in the project root for
 # license information.
-# --------------------------------------------------------------------------
+# ------------------------------------------------------------------------
 from .credentials import (
     AsyncCertificateCredential,
     AsyncClientSecretCredential,
     AsyncEnvironmentCredential,
+    AsyncImdsCredential,
     AsyncManagedIdentityCredential,
+    AsyncMsiCredential,
     AsyncTokenCredentialChain,
 )
 
+
+class AsyncDefaultAzureCredential(AsyncTokenCredentialChain):
+    """default credential is environment followed by MSI/IMDS"""
+
+    def __init__(self, **kwargs):
+        super().__init__(AsyncEnvironmentCredential(**kwargs), AsyncManagedIdentityCredential(**kwargs))
+
+
 __all__ = [
     "AsyncCertificateCredential",
     "AsyncClientSecretCredential",
+    "AsyncDefaultAzureCredential",
     "AsyncEnvironmentCredential",
+    "AsyncImdsCredential",
     "AsyncManagedIdentityCredential",
+    "AsyncMsiCredential",
     "AsyncTokenCredentialChain",
 ]
```

Comment:
Does it make sense to export these without the Async prefix, so that the same credential names can be used in both the sync and async?

Reply:
They could just have the same names, if we require a qualified import:

sync
>```py
>from azure.identity import ClientSecretCredential
>```
async
>```py
>from azure.identity.aio import ClientSecretCredential
>```

Which is fine by me. I followed the `azure-core` policy naming pattern here but I'm not attached to it.